upi/metal/main.tf

```diff
@@ -98,14 +100,16 @@ resource "matchbox_group" "worker" {
 provider "packet" {}
 
 locals {
-  packet_facility = "sjc1"
+  # The packet facility to deploy to - core facilities:
+  # all,ams1,iad2,dc13,dfw2,ewr1,ny5,sin3,sjc1,nrt1
+  packet_facility = "any"
```

Comment:
the nodes in the same cluster need to be in same facility because of network latency concerns.

Reply:
Glad we agree here... which is why we should actually use this local variable to set the facility... right now master is set to provision the instances wherever, which can lead to masters, workers and bootstrap distributed around the world:

https://github.com/openshift/installer/blob/1da6208796da9e0fa4cd8a4fa1467094342c6168/upi/metal/main.tf#L108

https://github.com/openshift/installer/blob/1da6208796da9e0fa4cd8a4fa1467094342c6168/upi/metal/main.tf#L121

https://github.com/openshift/installer/blob/1da6208796da9e0fa4cd8a4fa1467094342c6168/upi/metal/main.tf#L146

Since I did not want to change "any" to something specific and break CI because there is no availability... I chose to change the variable to any for now - I can change it back to sjc1 if you think there's enough availability?

Reply:
let's use the local and keep it set to any...